app/_src/gateway-operator/production/upgrade/data-plane/blue-green.md

```diff
@@ -0,0 +1,173 @@
+              env:
+              - name: KONG_LOG_LEVEL
+                value: debug
+              readinessProbe:
+                initialDelaySeconds: 1
+                periodSeconds: 1
+    ```
+
+    > **NOTE**: Currently only `BreakBeforePromotion` is available as promotion strategy.
+
+    When applied like this, {{ site.kgo_product_name }} will deploy new `Services` through which you'll be able to access new `Pod`s once they are available.
+
+    By default no `Pod`s will be deployed immediately, instead {{ site.kgo_product_name }} will observe the `DataPlane` resource for `spec` changes and when any configuration drift is detected it will spawn a new "preview" `Deployment` which will contain the changes applied to the `DataPlane` resource.
+
+1. Wait for `DataPlane` to be ready to accept changes
+
+    ```bash
+    kubectl wait dataplane bluegreen --for=jsonpath='{.status.rollout.conditions[*].reason}'=WaitingForChange
+    ```
+
+1. Test it out by patching the `DataPlane` with a new `image`:
+
+    ```bash
+    $ kubectl patch dataplane bluegreen --type='json' -p='[{"op": "replace", "path": "/spec/deployment/podTemplateSpec/spec/containers/0/image", "value":"kong:3.3.1"}]'
+    dataplane.gateway-operator.konghq.com/bluegreen patched
+    ```
+
+    After this patch gets applied you'll be able to access the new {{ site.base_gateway }} `Pod`s via the "preview" ingress `Service`.
+
+1. To find the "preview" `Service` you can look up `DataPlane` status, and more specifically its `rollout` field:
+
+    ```bash
+    $ kubectl get dataplane bluegreen -o jsonpath-as-json='{.status.rollout}
+    [
+        {
+            "conditions": [
+                {
+                    "lastTransitionTime": "2023-09-21T11:40:25Z",
+                    "message": "",
+                    "observedGeneration": 2,
+                    "reason": "AwaitingPromotion",
+                    "status": "False",
+                    "type": "RolledOut"
+                }
+            ],
+            "deployment": {
+                "selector": "6cf0d993-2319-43d5-bfdc-e2cadd6bd7e3"
+            },
+            "services": {
+                "adminAPI": {
+                    "addresses": [
+                        {
+                            "sourceType": "PrivateIP",
+                            "type": "IPAddress",
+                            "value": "None"
+                        }
+                    ],
+                    "name": "dataplane-admin-bluegreen-cx6nq"
+                },
+                "ingress": {
+                    "addresses": [
+                        {
+                            "sourceType": "PrivateLoadBalancer",
+                            "type": "IPAddress",
+                            "value": "172.18.0.101"
+                        },
+                        {
+                            "sourceType": "PrivateIP",
+                            "type": "IPAddress",
+                            "value": "10.96.28.2"
```

Comment:
https://datatracker.ietf.org/doc/html/rfc5737 use the IP address reserved for documentation